packages/demo-app/src/app/app.tsx

```diff
@@ -33,12 +33,6 @@ const GlobalStyle = createGlobalStyle`
     width: 100%;
   }
 
-  .main-container {
```

Comment:
why do we need to change that?

Reply:
I'm removing dead CSS code (at least, it's the objective)

Reply:
well I think it should be done in the demo move to the doc PR...

Reply:
I didn't intend to break the demo-app if it's why you assume. I can definitely revert.